Makefile

```diff
@@ -196,17 +197,34 @@ build-image:
 run: build-image
 	@CID=$(shell docker run --net=host -d --name $(SERVER) $(IMAGE)) && echo $${CID}
 
+test-cli:
+	@for pkg in $(CLI_TEST_PACKAGES) ; do \
+		go test $$pkg ; \
+	done
+
 test-unit:
 	@for pkg in $(UNIT_TEST_PACKAGES) ; do \
 		go test $$pkg ; \
 	done
 
 test-integration:
+	@docker service rm amp-integration-test || true
```

Comment:
This is slightly misleading. Even when the test runs correctly, it returns an error. 

`Error response from daemon: service amp-integration-test not found`